### DISCUSSION TITLE
Cannot render sample table in react-markdown

### DISCUSSION THREAD

**Discussion by poikjo** [remarkjs/remark#1232] [Q&A] [answered]
I'm trying to render samples (as also my own markdown) to render table in react-markdown but immediately when I add table to md the render fails on exception: "TypeError: Cannot set properties of undefined (setting 'inTable')"... from this module mdast-util-gfm-table.

<img width="306" alt="image" src="https://github.com/remarkjs/remark/assets/17199477/69b7d9f5-8fb0-47b8-89cd-c77ae1788546">

I feel this must be an user error but cannot find what could cause it. Any advices where to look at?

**Comment by ChristianMurphy** [comment, marked as answer]
Duplicate of https://github.com/remarkjs/remark-gfm/issues/57

To reiterate:
A major version release is rolling out across all: `unified`, `remark`, `rehype`, and `micromark` packages.
This includes both type changes (static analysis error) and changes to the parser (potential runtime error) if the major versions are mismatched.
For now pin the previous major of `remark-gfm` until the new `react-markdown` major is released .
